File: strategies/php/strategy_binance_order.php
<?php
/**
 * Strategy: Binance Order
 * Language: PHP
 * Exchange: Binance (Spot)
 */
require 'vendor/autoload.php';
use Binance\API;

$api = new API('KEY','SECRET');
$order = $api->order([
    'symbol'=>'BTCUSDT','side'=>'BUY','type'=>'LIMIT','quantity'=>0.01,'price'=>29000
]);
print_r($order);

File: strategies/dart/strategy_binance_order.dart
// Strategy: Binance Order
// Language: Dart
// Exchange: Binance (Spot)
import 'package:binance/binance.dart';

void main() async {
  final client = Binance(apiKey: 'KEY', secret: 'SECRET');
  final order = await client.createOrder('BTCUSDT', 'BUY', 'LIMIT', '0.01', price:'29000');
  print(order);
}
